src/cljc/rems/common/application_util.cljc

```diff
@@ -27,3 +29,26 @@
 
 (defn is-handler? [application user]
   (contains? (workflow-handlers application) user))
+
+(defn- str-to-int [s]
+  #?(:clj (Integer/parseInt s)
+     :cljs (js/parseInt s 10)))
+
+;; (defn- printToConsole [s]
+;;   #?(:clj (println s)
+;;      :cljs (js/console.log s)))
+
+(defn parse-sortable-external-id
+  "The idea is to parse all numbers from the string to vector and then compare the values in the vector"
```

Comment:
This is the wrong idea in that it doesn't handle now the possible differing string parts?